### PR TITLE
chore(SF-0.131): downport consolidates API of ObjectPage and DynamicPage

### PR DESCRIPTION
**Pull request by Lukas742**
BREAKING CHANGE: DynamicPageHeader: children are no longer displayed as flex items to support other display types like grid. To align children you now need to add the container (like FlexBox) and CSS yourself.
BREAKING CHANGE: DynamicPageTitle: subHeading has been renamed. Please use subheading instead.
BREAKING CHANGE: DynamicPage: header has been renamed. Please use headerContent instead.
BREAKING CHANGE: DynamicPage: title has been renamed. Please use headerTitle instead.
BREAKING CHANGE: ObjectPage: title has been renamed to headerTitle and is now defining the upper, static, title section of the ObjectPage. It expects to receive the DynamicPageTitle component.
BREAKING CHANGE: ObjectPage: noHeader has been removed. It is now sufficient not to set headerTitle and header to achieve the same behavior.
BREAKING CHANGE: ObjectPage: title, subTitle, headerActions, breadcrumbs and keyInfos should now be passed to the corresponding DynamicPageTitle props.
BREAKING CHANGE: ObjectPageSection: title and titleUppercase has been renamed. Please use heading and headingUppercase instead.
BREAKING CHANGE: ObjectPageSubSection: title has been renamed. Please use heading instead.
BREAKING CHANGE: getScrollBarWidth has been removed.

### CODE REVIEW

File: packages/base/src/dist/Utils.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { deprecationNotice, enrichEventWithDetails } from '../utils';
```
Comment (MarcusNotheis) [inline packages/base/src/dist/Utils.ts:1]: I think we don't have a dist folder for this release?

File: packages/main/src/components/DynamicPage/index.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { enrichEventWithDetails } from '@ui5/webcomponents-react-base/dist/Utils';
```
Comment (MarcusNotheis) [inline packages/main/src/components/DynamicPage/index.tsx:1]: This should import from `lib`:
```suggestion
import { enrichEventWithDetails } from '@ui5/webcomponents-react-base/lib/Utils';
```

**Review by MarcusNotheis — CHANGES_REQUESTED** [review]

File: packages/base/src/dist/Utils.ts (hunk 1 above)
Comment (Lukas742) [reply packages/base/src/dist/Utils.ts]: Done

File: packages/main/src/components/DynamicPage/index.tsx (hunk 2 above)
Comment (Lukas742) [reply packages/main/src/components/DynamicPage/index.tsx]: Done

**Review by MarcusNotheis — APPROVED** [review]

**Pull request merged**